editor/multi_node_edit.cpp

```diff
@@ -152,7 +152,12 @@ void MultiNodeEdit::_get_property_list(List<PropertyInfo> *p_list) const {
 				datas.push_back(usage.getptr(F->get().name));
 			}
 
-			usage[F->get().name].uses++;
+			// Make sure only properties with the same exact PropertyInfo data will appear
+			if (usage[F->get().name].info.type == F->get().type &&
+					usage[F->get().name].info.hint == F->get().hint &&
+					usage[F->get().name].info.hint_string == F->get().hint_string &&
+					usage[F->get().name].info.usage == F->get().usage)
```

Comment:
Before this is merged, I'd like to ask if I should leave the comparison this way, or if I should move it to an `operator==` function inside PropertyInfo, or if perhaps I should truncate the comparisons down to just the type comparison.

Also, I tried using `.operator Dictionary()` on both sides and comparing the two dictionaries, but that ended up never showing any of the properties. I don't know what's going on there.